ci/benchmarks/test_benchmarks.py

```diff
@@ -0,0 +1,213 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import logging
+import pytest
+import pandas
+import modin.pandas as pd
+from modin.pandas.test.utils import random_state
+
+RAND_LOW = 0
+RAND_HIGH = 100
+
+
+class weakdict(dict):
+    __slots__ = ("__weakref__",)
+
+
+# TODO: Using weak reference cache results in data object being always
+# collected which defeats the purpose of a cache. Maybe there is some
+# workaround for this which would allow objects to remain in
+# memory. Currently usage of weak reference dictionaries is turned off
+# because it slows benchmark down since objects have to be always
+# generated again.
+#
+# import weakref
+# data_cache = weakref.WeakValueDictionary()
+# dataframes_cache = weakref.WeakValueDictionary()
+
+data_cache = dict()
+dataframes_cache = dict()
+
+
+def gen_int_data(ncols, nrows, rand_low, rand_high):
+    cache_key = ("int", ncols, nrows, rand_low, rand_high)
+    if cache_key in data_cache:
+        return data_cache[cache_key]
+
+    logging.info(
+        "Generating int data {}x{} [{}-{}]".format(ncols, nrows, rand_low, rand_high)
```

Comment:
I think as the first dimension should be `nrows` and as the second one should be `ncols`.

Reply:
Why? Cols*rows is X*Y dimensions.

Reply:
Yes, but when we call `df.shape`, we see row count as the first number and column count as the second number. An example:
```python
import modin.pandas as pd
df = pd.DataFrame([[1,1,1],[1,1,1]])
df
   0  1  2
0  1  1  1
1  1  1  1
df.shape
(2, 3)
```
That's why I think, when we write something like this "data ...x...", it's better to use row count as the first number.

Reply:
Ok I changed log output to remove any ambiguity